public/constant/content.ts

```diff
@@ -1,29 +1,33 @@
 const common = {
   ethTaipei: "ETHTaipei 2024 | March 21-24",
   ethTaipeiIntro:
-    "Welcome to the ETHTaipei 2024 event held in Taiwan. With a thriving Ethereum and developer community in Taiwan, ETHTaipei 2024 has brought together teams from around the world to participate in a 3-day hackathon and 2-day conference focusing on the application and technology of Ethereum.",
+    // "Welcome to the ETHTaipei 2024 event held in Taiwan. With a thriving Ethereum and developer community in Taiwan, ETHTaipei 2024 has brought together teams from around the world to participate in a 3-day hackathon and 2-day conference focusing on the application and technology of Ethereum.",
+    "Welcome to the ETHTaipei 2024 event held in Taiwan. With a thriving Ethereum and developer community in Taiwan, ETHTaipei 2024 has brought together teams from around the world to participate in a 2-day conference focusing on the application and technology of Ethereum.",
```

Comment:
TODO: Update description to 2025 as well

Reply:
done! next time if possible pls help fix it 🙏 😁